exasol/error/_error.py

```diff
@@ -93,35 +100,58 @@ def _long(error: Error):
         return output[:-1]
 
 
+@dataclass(frozen=True)
+class StaticError:
+    identifier: str
+    message: str
+    messagePlaceholders: List[Dict[str, str]]
+    description: Optional[str]
+    mitigations: List[str]
+    sourceFile: str
+
+    def to_dict(self) -> Dict[str, Any]:
```

Comment:
You may be able to use dataclasses.asdict https://docs.python.org/3/library/dataclasses.html#dataclasses.asdict, depends if you need the specific handling of the description

Reply:
Good idea. Using `dataclasses.asdict(self)` and then removing `description` if None.